examples/jobs/dp-bert-launch-job.md

```diff
@@ -12,15 +12,17 @@ The Python script `dp_bert_large_hf_pretrain_hdf5.py` in head node is used as th
 Following are steps for launching a training job:
 
 1. In a terminal of the head node, activate the Python virtual environment: `source ~/aws_neuron_venv_pytorch_p37/bin/activate`.
-2. Download a shell run script to head node to set up the runtime environment: 
+
+2. Download dataset. In a terminal of the head node, follow [instructions in Neuron documentation](https://awsdocs-neuron-staging.readthedocs-hosted.com/en/release_2.3.0rc2/frameworks/torch/torch-neuronx/tutorials/training/bert.html#downloading-tokenized-and-sharded-dataset-files)
```

Comment:
Move this section into step #2 in "there will be three scripts required". Also change it to "there will be three scripts required in addition to dataset files"